src/rhsmlib/services/register.py

```diff
@@ -128,5 +130,8 @@ def validate_options(self, options):
             elif options['environment']:
                 raise exceptions.ValidationError(_("Error: Activation keys do not allow environments to be"
                     " specified."))
+        elif options.get('jwt_token') is not None:
+            # TODO: add more checks here
```

Comment:
Is this TODO for another PR?

Reply:
I don't know what was my intention with this TODO. We can delete.